### PR TITLE
Reintegrate cluster binaries to top-level project

### LINKED ISSUE #1691: Reintegrate `cluster` binaries to top-level project

### PR DESCRIPTION
**Pull request by bquenin**
Fix #1691 

## How to test
```
ampmake clean build
amp cluster create
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -75,8 +75,7 @@ clean: clean-protoc cleanall-cli clean-server clean-beat clean-agent clean-monit
 DOCKER_CMD ?= "docker"
 
 build-base: protoc build-server build-gateway build-beat build-agent build-monit
-build-local-plugin: build-amp-local build-ampagent
```
Comment (ndegory) [inline Makefile:78]: this target was used by the Travis build, no need to build the aws plugin (and soon the azure plugin, etc) when we can only test locally.

Comment (bquenin) [reply Makefile:78]: even if we're not actually using them, it's still good to make sure they actually build

Comment (ndegory) [reply Makefile:78]: sure, but the build time is increasing significantly

**Review by ndegory — CHANGES_REQUESTED** [review]
```
make VERSION=latest build-ampagent
make build-ampagent
docker run --rm appcelerator/ampagent:latest --help | grep version
2017/10/07 00:22:23 ampctl (version: latest, build: 18ddddef)
# this is OK
docker run --rm appcelerator/ampagent:0.16.0-dev --help | grep version
2017/10/07 00:22:45 ampctl (version: latest, build: 18ddddef)
# this is not OK

On master:
make build-amp-agent
docker run --rm appcelerator/ampagent:0.16.0-dev --help | grep version
2017/10/07 00:24:36 ampctl (version: 0.16.0-dev, build: 68d5f96f)
```
we have to make sure that the build of the binary is done with the right tag, and not from the cache.

**Comment by ndegory** [comment]
sorry for the header style in the review, it was supposed to be a commented line...

**Review by ndegory — APPROVED** [review]

**Comment by ndegory** [comment]
still the issue with the `make VERSION=latest build-ampagent`, but that should be fixed when the multi staged build will be implemented

**Comment by bquenin** [comment]
For reference #1694

**Pull request merged**